tendrl/monitoring_integration/objects/config/__init__.py

```diff
@@ -0,0 +1,16 @@
+from tendrl.commons import config as cmn_config
+from tendrl.commons import objects
+
+
+class Config(objects.BaseObject):
+    internal = True
+
+    def __init__(self, config=None, *args, **kwargs):
+        self._defs = {}
+        super(Config, self).__init__(*args, **kwargs)
+
+        self.data = config or cmn_config.load_config(
+            'monitoring-integration',
+            "/etc/tendrl/node-agent/node-agent.conf.yaml"
```

Comment:
https://github.com/Tendrl/monitoring-integration/blob/master/etc/tendrl/monitoring-integration/monitoring-integration.conf.yaml.sample
I have created a sample conf file in etc/tendrl/monitoring_integration. but its doesn't have etcd port and etcd host, which is required to set up etcd connection. I was fetching this from  https://github.com/Tendrl/monitoring-integration/pull/26/files/4d257a639474f3bc1ffc4ecdff01f16df4fdab80#diff-a850d8aa3fc778cea76799c0cb8c1df9R14